rxbinding/src/androidTest/java/com/jakewharton/rxbinding/widget/RxAutoCompleteTextViewTest.java

```diff
@@ -82,4 +88,61 @@
 
     o.assertNoMoreEvents();
   }
+
+  @TargetApi(Build.VERSION_CODES.JELLY_BEAN)
```

Comment:
You also need `@SuppressSdk` with `JELLY_BEAN` so that the test runner skips this test on pre-JB.